### ISSUE TITLE
Statusy

### ISSUE THREAD

**Issue by wontroba666** [PayU/plugin_magento_2#17]
Witam, jak włączyć statusy w module? czy opłącę czy nie opłacę operacje nie są przypisywane do żadnych statusów, a to chyba nie możliwe żeby oficjalne rozszerzenie tak działało

**Comment by regdos** [comment]
Statusy powinny się same zmieniać. Czy strona działa na publicznym adresie?

**Comment by wontroba666** [comment]
tak, strona działa na publicznym adresie. w przypadku poprawnej płatności PAYu, pole ze statusem jest puste, nie jest wysyłany e-mail potwierdzajacy
